ui/packages/components/src/types/replay.ts

```diff
@@ -0,0 +1,13 @@
+import durationPlugin from 'dayjs/plugin/duration';
+
+const replayStatuses = ['CREATED', 'ENDED'] as const;
+export type ReplayStatus = (typeof replayStatuses)[number];
+
+export type Replay = {
+  name: string;
+  status: ReplayStatus;
+  createdAt: Date;
+  endedAt?: Date;
+  duration?: durationPlugin.Duration;
```

Comment:
If we're using dayjs for this, shouldn't we add a linear task to replace the date utils by it? https://github.com/inngest/inngest/blob/14baa56e6fb74cc555465f84d2d642b3ef679d93/ui/packages/components/src/utils/date.ts#L21C17-L21C35

Reply:
Added it https://linear.app/inngest/issue/INN-2389/use-consistent-datetime-utils-across-apps-and-libraries :)